### PR TITLE
Add Darcy Clarke

### PR DESCRIPTION
**Pull request by darcyclarke**

### CODE REVIEW

**Comment by arthurvr** [comment]
Please don't create a new PR, instead update the existing one. This is unnecessary noise and clutters our issue tracker ;)

File: readme.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Amas is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym).
 - [Maxime Euzière](https://github.com/xem/ama) - Front-end hacker and JS code-golfer.
 - [Robert Haritonov](https://github.com/operatino/ama) - Front-end Team Lead, [SourceJS](http://sourcejs.com) maintainer and Living Style Guide evangelist.
 - [Lim Chee Aun](https://github.com/cheeaun/ama) - Product engineer. GitHub stargazer. Anime lover.
+- [Darcy Clarke](https://github.com/darcyclarke/ama) - Developer, Designer, Founder, Mentor & Speaker
```
Comment (arthurvr) [inline readme.md]: End your bio with a period.

Comment (darcyclarke) [inline readme.md]: It was a patch branch so I just deleted it. Appreciate the thoroughness

Comment (arthurvr) [inline readme.md]: The fact that something is a patch branch doesn't mean you can't edit it.

**Comment by arthurvr** [comment]
Merged. Thanks.

**Comment by sindresorhus** [comment]
Hey. Want to ask you a favor. Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=
https://github.com/sindresorhus/ama/issues?q=

**Pull request closed without merging**